File: 1. Two Sum/solution.java
import java.util.Scanner;

class solution {
    public static int[] findTwoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length - 1; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[]{i, j};
                }
            }
        }
        return new int[0]; // This line is theoretically unreachable as per problem statement
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        System.out.print("Enter the array size: ");
        int n = in.nextInt();
        
        int[] nums = new int[n];
        System.out.println("Enter " + n + " array elements:");
        for (int i = 0; i < n; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            nums[i] = in.nextInt();
        }
        
        System.out.print("Enter the target sum: ");
        int target = in.nextInt();
        
        int[] result = findTwoSum(nums, target);
        
        System.out.println("Indices: [" + result[0] + ", " + result[1] + "]");
        
        in.close();
    }
}

File: 1. Two Sum/solution.py
def find_two_sum(nums, target):
    for i in range(len(nums) - 1):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []  # This line is theoretically unreachable as per problem statement

def main():
    n = int(input("Enter the array size: "))
    nums = []
    for i in range(n):
        num = int(input(f"Enter element {i + 1}: "))
        nums.append(num)
    target = int(input("Enter the target sum: "))
    result = find_two_sum(nums, target)
    print(f"Indices: [{result[0]}, {result[1]}]")

if __name__ == "__main__":
    main()
